### PR TITLE
Add exporterhelper to wrap all exporters with observability.

### PR DESCRIPTION
**Pull request by bogdandrutu**
Only Logging and OpenCensus exporters changed in this PR, I will update all the others in a followup PR.

### CODE REVIEW

File: exporter/exporterhelper/common.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	var opts ExporterOptions
+	for _, op := range options {
+		op(&opts)
+	}
+	return opts
+}
+
+func errToStatus(err error) trace.Status {
+	if err != nil {
+		return trace.Status{Code: trace.StatusCodeUnknown, Message: err.Error()}
```
Comment (songy23) [inline exporter/exporterhelper/common.go]: Should we check if `err` is an RPC error and set precise status here?

File: exporter/exporterhelper/metricshelper.go [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+	"github.com/census-instrumentation/opencensus-service/observability"
+	"go.opencensus.io/trace"
+
+	"github.com/census-instrumentation/opencensus-service/data"
+	"github.com/census-instrumentation/opencensus-service/exporter"
+)
+
+const (
+	numDroppedMetricsAttribute  = "num_dropped_metrics"
```
Comment (songy23) [inline exporter/exporterhelper/metricshelper.go]: Consider putting these in `constants.go`?

File: exporter/exporterhelper/tracehelper.go [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+	"go.opencensus.io/trace"
+
+	"github.com/census-instrumentation/opencensus-service/data"
+	"github.com/census-instrumentation/opencensus-service/exporter"
+	"github.com/census-instrumentation/opencensus-service/observability"
+)
+
+const (
+	numDroppedSpansAttribute  = "num_dropped_spans"
```
Comment (songy23) [inline exporter/exporterhelper/tracehelper.go]: Same here.

File: exporter/exporterhelper/common.go (hunk 1 above)
Comment (bogdandrutu) [reply exporter/exporterhelper/common.go]: Added a TODO for the moment.

File: exporter/exporterhelper/common.go [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+// Construct the ExporterOptions from multiple ExporterOption.
+func newExporterOptions(options ...ExporterOption) ExporterOptions {
+	var opts ExporterOptions
+	for _, op := range options {
+		op(&opts)
+	}
+	return opts
+}
+
+func errToStatus(err error) trace.Status {
```
Comment (pjanotti) [inline exporter/exporterhelper/common.go]: except for this function and the related var it seems that a better name will be options.go, then the errToStatus and the var perhaps can be moved to other file

Comment (bogdandrutu) [reply exporter/exporterhelper/common.go]: Will do that in a different PR. Thanks for the suggestion.

**Pull request merged**